usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -26,38 +26,37 @@ def covid_def_code_strings():
 
 def latest_gtas_of_each_year_queryset():
     q = Q()
-    for final_for_fy in finals_for_fy(False):
-        q |= Q(fiscal_year=final_for_fy[0]) & Q(fiscal_period=final_for_fy[1])
-    for final_for_fy in finals_for_fy(True):
-        q |= Q(fiscal_year=final_for_fy[0]) & Q(fiscal_period=final_for_fy[1])
+    for final_for_fy in finals_for_all_fy():
+        q |= Q(fiscal_year=final_for_fy[0]) & Q(fiscal_period=final_for_fy[2])
```

Comment:
Up to you if you want to "fix" this, but GTAS only has monthly data so there's no sense querying quarters.  Didn't notice that before.  A simple `if not final_for_fy[1]:` before the `q |=` should eliminate them.

Reply:
Wrong PR, look at #2567 

I thought about quarters not being separate from monthly in GTAS, idea makes sense

Reply:
Done